include/uaclient/uasession.h

```diff
@@ -86,8 +87,14 @@ class UaSession
             UaStatusCodeArray &     results,
             UaDiagnosticInfos &     diagnosticInfos );
 
+    UaStatus disconnect(
+            ServiceSettings &       serviceSettings,
```

Comment:
In some places Hungarian notation is used and in some other not. Is there any reason for not keeping it uniform?

Reply:
Yes I noticed this as well... and I'm not in favour of Hungarian notation... but part of it comes from the UA API itself which we want to follow, so it is not really possible to completely avoid it.